repose-aggregator/components/filters/versioning-filter/src/main/java/org/openrepose/filters/versioning/util/ContentTransformer.java

```diff
@@ -52,26 +49,19 @@
         XSLT_TRANSFORMER_FACTORY.setErrorListener(new LogErrorListener());
     }
 
-    @Deprecated
-    private final Transform<JAXBElement, String> jsonTransform;
-    @Deprecated
-    private final Transform<JAXBElement, String> xmlTransform;
     private final StreamTransform<JAXBElement, OutputStream> jsonStreamTransform;
     private final StreamTransform<JAXBElement, OutputStream> xmlStreamTransform;
 
     public ContentTransformer() {
         try {
             final JAXBContext context = JAXBContext.newInstance(
-                    org.openrepose.filters.versioning.schema.ObjectFactory.class,
-                    org.openrepose.filters.versioning.config.ObjectFactory.class);
+                    org.openrepose.filters.versioning.config.ObjectFactory.class,
+                    org.openrepose.filters.versioning.schema.ObjectFactory.class);
```

Comment:
There wasn't any. Initially, I dropped the schema package thinking it was redundant with the config package. Alas, not so. The schema package is used to form responses, while the config package is used to configure the filter. When I add that package back to the JAXB context, I did so manually.

So I'll flip these back before this gets merged.